src/styles/themes/input.ts

```diff
@@ -0,0 +1,225 @@
+import { colors } from './colors'
+import { fonts } from './fonts'
+export type Input = typeof input
+
+export const input = {
+  chat: {
+    large: `
+        &>input: {
+            height: 48px;
+            width: 639px;
+            padding: 12px 60px 12px 20px;
+
+            &::placeholder{
+               ${fonts.body01R};
+                color: ${colors.grayScale50};
+            }
+            $:hover{
+                ${fonts.body01R}
+                color: ${colors.grayScale90};'
+            }
+            &:focus{
+                ${fonts.body01R}
+                color: ${colors.grayScale90};'
+            }
+        }
+    }`,
+
+    small: `
+        &>input: {
+            height: 40px;
+            width: 328px;
+            padding: 10px 52px 10px 12px;
+
+            &::placeholder{
+                ${fonts.body02R};
+                 color: ${colors.grayScale50};
+            }
+            $:hover{
+                ${fonts.body02R};
+                 color: ${colors.grayScale90};
+            }
+            $:focus{
+                ${fonts.body02R};
+                color: ${colors.grayScale90};
+            }
+        },
+    `
+  },
+  default: {
+    large: {
+      default: `
+        &>input: {
+            margin: 8px 0;
+            width: 328px;
+            height: 54px;
+            border: none;
+            padding: 16px 12px 16px 12px;
+            &::placeholder{
+                ${fonts.subtitle01M};
+                color: ${colors.grayScale50};
+            }
+            $:hover{
+                ${fonts.subtitle01M};
+                color:${colors.grayScale90}
+            }
+            $:focus{
+                ${fonts.subtitle01M};
+                color:${colors.grayScale90}
```

Comment:
```suggestion
                color:${colors.grayScale90};
            }
            $:focus{
                ${fonts.subtitle01M};
                color:${colors.grayScale90};
```
; 없으면 해당 테마 이용 시, 이후 속성들이 다 무시되어서 붙여주어야할 것 같아요!